Web_Development/FrontEnd/HTML_Tables/HTML Table Attributes.md

```diff
@@ -0,0 +1,131 @@
+VALIGN=TOP|BOTTOM|MIDDLE>
+<TH ALIGN=LEFT|RIGHT| CENTER|MIDDLE|BOTTOM
+VALIGN=TOP|BOTTOM|MIDDLE>
+```
+5.   **Width & Height**: Width attribute specifies the width of a table and height adjusts its height. If you want to set width and height of a table then you should use width/height attribute in table tag.
+```
+<TH WIDTH=? HEIGHT=?> – (in pixels)
+<TH WIDTH="%" HEIGHT="%"> – (percentage of table)
+<TABLE WIDTH=? HEIGHT=?> – (in pixels)
+<TABLE WIDTH="%" HEIGHT="%"> – (percentage of page)
+<TD WIDTH=? HEIGHT=?> - (in pixels)
+<TD WIDTH="%" HEIGHT="%"> – (percentage of table)
+```
+6. **Bgcolor**: Bgcolor attribute specifies the background color of a table. If you want to use a background color in your table then you should use bgcolor attribute in table tag.
+```
+<TH BGCOLOR="#$$$$$$">
+<TD BGCOLOR="#$$$$$$">
+```
+7.   **Colspan and rowspan**:You will use colspan attribute if you want to merge two or more columns into a single column. Similar way you will use rowspan if you want to merge two or more rows.
+```
+<td rowspan =?></td>
+<td colspan =?></td>
+```
+8.  **Caption tag**:The caption tag will serve as a title or explanation for the table and it shows up at the top of the table. This tag is deprecated in newer version of HTML/XHTML.
+```
+<caption>This is the caption</caption>
+```
+## Table Header, Body, and Footer
+Tables can be divided into three portions − a header, a body, and a foot. The head and foot are rather similar to headers and footers in a word-processed document that remain the same for every page, while the body is the main content holder of the table.
+
+The three elements for separating the head, body, and foot of a table are −
+
+*   `<thead>` − to create a separate table header.
+
+*   `<tbody>` − to indicate the main body of the table.
+
+*   `<tfoot>` − to create a separate table footer.
+
+A table may contain several `<tbody>` elements to indicate different pages or groups of data. But it is notable that `<thead>` and `<tfoot>` tags should appear before `<tbody>`
+
+```
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>HTML Table</title>
+   </head>
+   <body>
+      <table border = "1" width = "100%">
+         <thead>
+            <tr>
+               <td colspan = "4">This is the head of the table</td>
+            </tr>
+         </thead>      
+         <tfoot>
+            <tr>
+               <td colspan = "4">This is the foot of the table</td>
+            </tr>
+         </tfoot>
+         <tbody>
+            <tr>
+               <td>Cell 1</td>
+               <td>Cell 2</td>
+               <td>Cell 3</td>
+               <td>Cell 4</td>
+            </tr>
+         </tbody>
+      </table>
+   </body>
+   ```
+![header,body,footer.png](https://th.bing.com/th/id/OIP.-2rsLlxIZioqtpsIw0Uf1AHaCY?pid=ImgDet&rs=1)
+
+## Reference table for quick review
+![attributes-of-table-tag-ev.png](https://i1.wp.com/www.edupointbd.com/wp-content/uploads/2019/06/attributes-of-table-tag-ev.png?resize=640%2C317)

```

Comment:
This could be color inverted, to give more readability.

Reply:
How can I invert the color of these two images can you guide me? @code-reaper08

Reply:
You can edit the image using photoshop, I think it could be done in mobile itself using Picsart.